lib/model/parameter/StatesTagsParameter.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:openfoodfacts/interface/Parameter.dart';
+import 'package:openfoodfacts/model/State.dart';
+
+/// States Tags as completed or to-be-completed. Filter combo mode is 'AND'.
```

Comment:
Again, can't tell what a "Filter combo mode" is

Reply:
@M123-dev Fair enough; I've changed the comment.